src/inverted-index.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable no-undef */
```

Comment:
Rather than silencing your `no-undef` rule, you should consider adding globals to your ESLint rules.